device.go

```diff
@@ -22,13 +22,19 @@ import (
 	"time"
 )
 
+// DevicePath is a representation of a USB port path
+// The path describes the physical port connections, ordered from Bus to Device.
+type DevicePath []int
```

Comment:
I don't think it's necessary, given that Port is an int. The type itself doesn't actually offer any functionality, nor does it automatically meet the expectations described in the comment. It still depends on the DeviceDesc below to implement it correctly. I'd much rather have a []int in the DeviceDesc with a longer description of what it represents.